common/scripts/lib/_helpers.sh

```diff
@@ -305,6 +305,18 @@ __set_secret_key() {
   export SECRET_KEY="$response"
 }
 
+_validate_ip() {
+  local ip=$1
+  local step=$2
+  local rx='([1-9]?[0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])'
+  if [[ $ip =~ ^$rx\.$rx\.$rx\.$rx$ ]]; then
+    return
+  else
+    __process_error "Invalid response, please enter valid IP address"
+    $step
```

Comment:
why is this required?
- this should be a simple function that returns true/false and the caller needs to decide what to with the return value. this implementation is too complex for a simple problem